README.md

```diff
@@ -15,3 +15,10 @@ dotrun
 
 And click on the server link that appears in the server logs.
 
+### Run locally with search functionality
+
+If you wish to run the site locally with search enabled, you'll need to pass a key. This key can be obtained by contacting webteam-dev@canonical.com
```

Comment:
I would suggest you contact a member on the irc channel as the key should be shared via pastebin.

Reply:
We don't want to encourage people outside the company to email the list and only our team can email it without approval.